components/collapse/collapseItem.js

```diff
@@ -17,22 +18,29 @@ class CollapseItem extends Component {
 
   render() {
     const {
+      className,
       id,
       isActive,
       title,
       children,
-      onClick, // eslint-disable-line no-unused-vars
       iconPosition,
-      labelProps = {},
-      ...otherProps
+      dataAttrs,
+      labelDataAttrs,
```

Comment:
are we sure that here we pass only dataAttrs for the label?

Reply:
Yes, we are. We pass this prop to `getDataAttributes` function and the function returns only data attributes

Reply:
👍